doc/using-energyplus-for-compliance/src/introduction/features-related-to-appendix-g-and-leed.tex

```diff
@@ -72,7 +72,10 @@ \subsection{Baseline Building Rotations}\label{baseline-building-rotations}
 
 \subsection{Completing LEED Forms from Tabular Reports}\label{completing-leed-forms-from-tabular-reports}
 
-The U.S. Green Building Council building certification system called Leadership in Energy and Environmental Design (LEED) Green Building Rating System™ includes Energy and Atmosphere Credit 1 Optimize Energy Performance. Credit 1 includes an option that requires a series of building energy simulations that follow the procedures of ASHRAE Standard 90.1 Appendix G Performance Rating Method. The~ LEED Summary report provides many of the simulation results required for the forms. The report can be produced by specifying LEEDSummary in Output:Table:SummaryReports which is also part of the AllSummary option.
+The U.S. Green Building Council building certification system called Leadership in Energy and Environmental Design (LEED) Green Building Rating System™ includes Energy and Atmosphere Credit 1 Optimize Energy Performance. Credit 1 includes an option that requires a series of building energy simulations that follow the procedures of ASHRAE Standard 90.1 Appendix G Performance Rating Method. The~ LEED Summary report provides many of the simulation results required for the forms. The report can be produced by specifying LEEDSummary in Output:Table:SummaryReports which is also part of the AllSummary option. 
+
+The EAp2-4/5 Performance Ration Method Compliance subtable has some special options in order to show end uses that are different than the end-uses that are typically used with EnergyPlus. By specifing the end-use subcategories of "Cooking" most commonly with the ElectricEquipment and GasEquipment objects, "Elevators and Escalators" most commonly with the ElectricEquipment object, "Industrial Process" most commonly with the ElectricEquipment and GasEquipment objects, "Interior Lighting - Process" most commonly with the Lights object, or "Fans - Parking Garage" most commonly with the Exterior:FuelEquipment object, those end uses are shown in the EAp2-4/5 Performance Ration Method Compliance subtable.
```

Comment:
Noticed a stray tilde, or is that suppsed to be there?  "The~ LEED Summary report provides . . . "